std/src/main/scala/algebra/std/set.scala

```diff
@@ -6,9 +6,8 @@ import algebra.ring.{BoolRng, Semiring}
 
 package object set extends SetInstances
 
-trait SetInstances {
+trait SetInstances extends cats.kernel.std.SetInstances {
   implicit def setLattice[A]: GenBool[Set[A]] = new SetLattice[A]
-  implicit def setPartialOrder[A]: PartialOrder[Set[A]] = new SetPartialOrder[A]
   implicit def setSemiring[A] = new SetSemiring[A]
   def setBoolRng[A] = new SetBoolRng[A]
```

Comment:
For the record, because it is not compatible with the above `Semiring` (they differ in the definition of `plus`, "union" vs. "symmetric difference"). And I see that is not obvious at all. We should document the meaning (in terms of `Set` operations) of Ring operations.